var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -39,9 +40,14 @@ def install(self, spec, prefix):
             make_defs += ['BUILD_LAPACK_DEPRECATED=1']
 
         # Add support for OpenMP
-        # Note: Make sure your compiler supports OpenMP
         if '+openmp' in spec:
-            make_defs += ['USE_OPENMP=1']
+            # Note: Apple's most recent Clang 7.3.0 still does not support OpenMP.
+            # What is worse, Openblas (as of 0.2.18) hardcoded that OpenMP cannot
+            # be used with any (!) compiler named clang, bummer.
+            if spec.satisfies('%clang'):
+                raise InstallError('OpenMP is not currently supported by Openblas with Clang! Install without +openmp')
```

Comment:
It'd say instead `OpenBLAS does not support OpenMP with clang` instead.

Yes, looks good.